app/src/main/java/com/edu/sun/oereminder/data/repository/TimeSheetRepositoryImpl.kt

```diff
@@ -19,19 +23,34 @@ class TimeSheetRepositoryImpl(
         }
     }
 
-    override fun getCheckedTimeRecords(
+    override fun getTimeRecords(
         from: Long,
         to: Long,
         callback: SourceCallback<List<TimeRecord>>
     ) {
         with(appExecutors) {
             diskIO.execute {
-                val timeRecords = local.getCheckedTimeRecords(from, to)
+                val timeRecords = local.getTimeRecords(from, to)
                 mainThread.execute { callback.onSuccess(timeRecords) }
             }
         }
     }
 
+    override fun getTimeRecord(date: Long, callback: SourceCallback<TimeRecord>) {
+        with(appExecutors) {
+            diskIO.execute {
+                val calendar = GregorianCalendar().from(date)
+                val timeRecords =
+                    local.getTimeRecords(calendar.firstMillisOfDay(), calendar.lastMillisOfDay())
+                        .firstOrNull()
+                mainThread.execute {
+                    if (timeRecords != null) callback.onSuccess(timeRecords)
+                    else callback.onError(Exception())
```

Comment:
Nếu không viết trên 1 dòng thì phải thêm ngoặc đầy đủ 
```suggestion
                    if (timeRecords != null){
                        callback.onSuccess(timeRecords)
                    } else {
                         callback.onError(Exception())
                     }
```